content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,280 @@ weight: 50
+    with `kubectl`.
+    
+    {{< text bash >}}
+    $ cd install/kubernetes/helm/istio
+    $ helm upgrade --set global.meshExpansion=true istio-system .
+    $ cd -
+    {{< /text >}}
+    
+    {{< text bash >}}
+    $ cd install/kubernetes/helm/istio
+    $ helm template --set global.meshExpansion=true --namespace istio-system . > istio.yaml
+    $ kubectl apply -f istio.yaml
+    $ cd -
+    {{< /text >}}
+    
+    You can either set the option on the command line, as in our examples, or add it to a `.yaml` values file and pass it to
+    the command with `--values`, which is the recommended approach when managing configurations with multiple options. You
+    can see some sample values files in your Istio installation's `install/kubernetes/helm/istio` directory and find out
+    more about customizing Helm charts in the [Helm documentation](https://docs.helm.sh/using_helm/#using-helm).
 
-*   Setup Internal Load Balancers (ILBs) for Kube DNS, Pilot, Mixer and Citadel. This step is specific to
-each cloud provider, so you may need to edit annotations. You can use an ILB based on Keepalived at
-[here](https://github.com/gyliu513/work/tree/master/k8s/charts/keepalived) for demo or test in case where
-the cloud provider or private cloud (for example IBM Cloud Private) doesn't have load balancer service
-support out of box.
+2.  Find the IP address of the Istio ingress gateway, as this is how the mesh expansion machines will access [Citadel](/docs/concepts/security/) and [Pilot](/docs/concepts/traffic-management/#pilot-and-envoy).
 
     {{< text bash >}}
-    $ kubectl apply -f @install/kubernetes/mesh-expansion.yaml@
+
+    $ GWIP=$(kubectl get -n istio-system service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    $ echo $GWIP
+    35.232.112.158
+
     {{< /text >}}
 
-*   Generate the Istio `cluster.env` configuration to be deployed in the VMs. This file contains
-the cluster IP address ranges to intercept.
+3.  Generate a `cluster.env` configuration to deploy in the VMs. This file contains the k8s cluster IP address ranges to
+    intercept and redirect via Envoy. You specify the CIDR range when you install Kubernetes as `servicesIpv4Cidr`.
+    Replace $MY_ZONE and $MY_PROJECT in the following example commands with the appropriate values to obtain the CIDR
+    after installation:
 
     {{< text bash >}}
-    $ export GCP_OPTS="--zone MY_ZONE --project MY_PROJECT"
-    $ @install/tools/setupMeshEx.sh@ generateClusterEnv MY_CLUSTER_NAME
+
+    $ ISTIO_SERVICE_CIDR=$(gcloud container clusters describe $K8S_CLUSTER --zone $MY_ZONE --project $MY_PROJECT --format "value(servicesIpv4Cidr)")
+    $ echo -e "ISTIO_CP_AUTH=MUTUAL_TLS\nISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR\n" > cluster.env
+
     {{< /text >}}
 
-    Here's an example generated file
+4.  Check the contents of the generated file. It should be similar to the following example:
 
     {{< text bash >}}
+
     $ cat cluster.env
```

Comment:
We told them we were generating cluster.env in step 3, though.  If anything I'd explicitly state "the contents of the generated cluster.env file", removing the command because "we don't mention it's the cluster.env file" makes it *less* obvious what file we want them to look at.